src/qibo/result.py

```diff
@@ -150,12 +157,17 @@ class MeasurementOutcomes:
     """Object to store the outcomes of measurements after circuit execution.
 
     Args:
+        measurements (qibo.gates.M): The measurement gates containing the measurements.
+        backend (qibo.backends.AbstractBackend): Backend used for the calculations. If not provided the :class:`qibo.backends.GlobalBackend` is going to be used.
+        probabilities (np.ndarray): Use these probabilities to generate samples and frequencies.
+        samples (np.darray): Use these samples to generate probabilities and frequencies.
+        nshots (int): Number of shots used for samples, probabilities and frequencies generation.
     """
 
     def __init__(
         self,
         measurements,
-        backend,
+        backend=None,
```

Comment:
As best practice in the other modules, the arg ``backend`` is usually the last one.

Reply:
I see, but then we will probably need to open two small PR in qibojit and qibolab to update them accordingly. The reason it was not defaulting to None from the beginning is that these object are not supposed to be exposed to the user. Usually, their construction is taken care of by the backend. I don't know, it looks a little bit too much effort just to change the order of a parameter that will probably never be seen by the user anyway.